docs/build-strict/README.md

```diff
@@ -0,0 +1,24 @@
+# Build Strict
```

Comment:
Maybe we should call this file `strict-mode` and change the title to `Strict mode` as well since it's the title of the item in the sidebar, what do you think?

Reply:
Change the file name or the repository name?

Reply:
the folder, the file has to be `README` :smile: